src/main/java/com/hubspot/jinjava/lib/filter/FilterLibrary.java

```diff
@@ -82,6 +82,7 @@ protected void registerDefaults() {
       Md5Filter.class,
       MinusTimeFilter.class,
       MultiplyFilter.class,
+      FormatNumberFilter.class,
```

Comment:
This should go higher up to maintain the alphabetic sort now that the name has changed.